configs/default/HGLR-HGLRCF411SX1280.config

```diff
@@ -0,0 +1,102 @@
+# Betaflight / STM32F411SX1280 (S4SX) 4.3.0 Jun 12 2022 / 14:56:52 (6d17f3f) MSP API: 1.44
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define USE_MAX7456
+#define USE_RX_EXPRESSLRS
+#define USE_RX_EXPRESSLRS_TELEMETRY
+#define USE_RX_SX1280
+#define RX_CHANNELS_AETR
+#define USE_RX_SPI
```

Comment:
USE_RX_SPI is added by cloud API (as it's working without the define with BETAFPVF4SX1280).

Reply:
It has been changed